outputs.md

```diff
@@ -282,10 +306,22 @@ Text file of generated bandpass solutions. The first column is the frequency cha
 
 ### \<obsids\>\_Beam_\<pol\>.sav <br />
 
-### \<obsids\>\_\<filter\>_Dirty_\<pol\>.fits / \<obsids\>\_\<filter\>\_Model_\<pol\>.fits / \<obsids\>\_\<filter\>\_Residual_\<pol\>.fits <br />
+### \<obsids\>\_\<filter\>\_Dirty\_\<pol\>.fits / \<obsids\>\_\<filter\>\_Model_\<pol\>.fits / \<obsids\>\_\<filter\>\_Residual_\<pol\>.fits <br />
+
+Fits files containing the dirty, model, and residual (dirty minus model) images. These images are constrained to be real for XX and YY instrumental polarizations. Units are Jy per sterradian.
```

Comment:
Since there are multiple sensible ways to make complex things real, can there be a comment about how that constraint is done (take real part of amplitude or something more sophisticated)?